src/components/program-progress/data/programProgressLoader.js

```diff
@@ -0,0 +1,19 @@
+import { queryLearnerProgramProgressData } from '../../app/data';
+import { ensureAuthenticatedUser } from '../../app/routes/data/utils';
+
+export default function makeProgramProgressLoader(queryClient) {
```

Comment:
[suggestion] Since we are relying on the loader to pre-seed the query cache to enable no loading states in the UI, the loader function is important to have test coverage so the expected queries are asserted to have happened. See other loader tests as examples.